src/PKSim.UI.Starter/CLIApplicationStartup.cs

```diff
@@ -0,0 +1,80 @@
+﻿using OSPSuite.Assets;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Core.Services;
+using OSPSuite.Presentation.Core;
+using OSPSuite.Presentation.Services;
+using OSPSuite.Utility.Container;
+using OSPSuite.Utility.Events;
+using OSPSuite.Utility.Exceptions;
+using OSPSuite.Utility.Format;
+using PKSim.CLI.Core.MinimalImplementations;
+using PKSim.Core;
+using PKSim.Core.Services;
+using PKSim.Infrastructure;
+using PKSim.Presentation;
+using System;
+using System.Threading;
+using OSPSuite.Presentation;
+
+namespace PKSim.UI.Starter;
+
+public class CLIApplicationStartup : ApplicationStartup
+{
+   static IContainer _container;
```

Comment:
why static?

Reply:
All the `ApplicationStartup` classes are the same. They subclass from a core class and implement static methods for bootstrapping. The static methods can only access static members.